src/pages/RequestCallPage.js

```diff
@@ -0,0 +1,171 @@
+import {Component} from 'react';
```

Comment:
`React` must be included

Reply:
Hey @marcaaron, sorry for the late reply! I noticed that this was broken and pushed a fix. It seems like it was broken when I merged a remote and local version of my branch.